dist/javascripts/moj.slot-picker.js

```diff
@@ -86,6 +91,11 @@
 
       this.$_el.on('click chosen', '.BookingCalendar-dateLink, .DateSlider-largeDates li', function(e) {
         e.preventDefault();
+        if(self.limitReached()) {
```

Comment:
I don't know if we have linting set up but this whole block could do with spaces around the brackets and dropping blocks onto their own lines